gridscale/resource_gridscale_k8s.go

```diff
@@ -0,0 +1,476 @@
+							Computed: true,
+						},
+						"port": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"security_zone_uuid": {
+				Type:        schema.TypeString,
+				Description: "Security zone UUID linked to PaaS service",
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+			},
+			"network_uuid": {
+				Type:        schema.TypeString,
+				Description: "Network UUID containing security zone",
+				Computed:    true,
+			},
+			"k8s_release": {
+				Type:         schema.TypeString,
+				Description:  "Release number of k8s service",
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			"service_template_uuid": {
+				Type:        schema.TypeString,
+				Description: "PaaS service template that k8s service uses.",
+				Computed:    true,
+			},
+			"node_pool": {
+				Type:        schema.TypeList,
+				Required:    true,
+				MaxItems:    1,
+				Description: `Node pool's specification.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Name of node pool",
+						},
+						"node_count": {
+							Type:        schema.TypeInt,
+							Description: "Number of worker nodes",
+							Required:    true,
+						},
+						"cores": {
+							Type:        schema.TypeInt,
+							Description: "Cores per worker node",
+							Required:    true,
+						},
+						"memory": {
+							Type:        schema.TypeInt,
+							Description: "Memory per worker node (in GiB)",
+							Required:    true,
+						},
+						"storage": {
+							Type:        schema.TypeInt,
+							Description: "Storage per worker node (in GiB)",
+							Required:    true,
+						},
+						"storage_type": {
+							Type:        schema.TypeString,
+							Description: "Storage type (one of storage, storage_high, storage_insane)",
+							Required:    true,
+							ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+								valid := false
+								for _, stype := range storageTypes {
+									if v.(string) == stype {
+										valid = true
+										break
+									}
+								}
+								if !valid {
+									errors = append(errors, fmt.Errorf("%v is not a valid storage type. Valid types are: %v", v.(string), strings.Join(storageTypes, ",")))
+								}
+								return
+							},
+						},
+					},
+				},
+			},
+			"usage_in_minute": {
+				Type:        schema.TypeInt,
+				Description: "Number of minutes that PaaS service is in use",
+				Computed:    true,
+			},
+			"current_price": {
+				Type:        schema.TypeFloat,
+				Description: "Current price of PaaS service",
+				Computed:    true,
+			},
+			"change_time": {
+				Type:        schema.TypeString,
+				Description: "Time of the last change",
+				Computed:    true,
+			},
+			"create_time": {
+				Type:        schema.TypeString,
+				Description: "Time of the creation",
```

Comment:
Make it "Time this service was created"... "Time of the creation" might inflame theological disputes :smile: